addons/test_mail/tests/test_mail_activity.py

```diff
@@ -148,6 +149,35 @@ def test_activity_dont_notify_no_user_change(self):
             activity.with_user(self.user_admin).write({'user_id': self.user_employee.id})
         self.assertEqual(activity.user_id, self.user_employee)
 
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_activity_summary_on_change(self):
```

Comment:
test_activity_summary_sync ?

I would also add a small docstring telling the test's purpose, like
```
Test summary from type is copied on activities if set (currently only in form-based onchange
```